packages/react-components/react-breadcrumb/src/components/Breadcrumb/Breadcrumb.types.ts

```diff
@@ -1,16 +1,40 @@
 import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
 
+/**
+ * Data shared between breadcrumb components
+ */
+export type BreadcrumbContextValue = Pick<BreadcrumbProps, 'size' | 'dividerType'>;
```

Comment:
```suggestion
export type BreadcrumbContextValue = Required<Pick<BreadcrumbProps, 'size' | 'dividerType'>>;
```

nit: Let's add `Required` to ensure that values are defined